docs/CallDirection.md

```diff
@@ -0,0 +1,11 @@
+# CallDirection
+
+Direction of the Call. `inbound` for Calls into FreeClimb, `outboundAPI` for Calls initiated via the REST API,  `outboundDial` for Calls initiated by the `OutDial` PerCL command.
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**value** | **str** | Direction of the Call. &#x60;inbound&#x60; for Calls into FreeClimb, &#x60;outboundAPI&#x60; for Calls initiated via the REST API,  &#x60;outboundDial&#x60; for Calls initiated by the &#x60;OutDial&#x60; PerCL command. |  must be one of ["inbound", "outboundAPI", "outboundDial", ]
+
```

Comment:
The format of this document doesn't correspond with the other sdk MD files for enums